src/components/CriteriaService/CriteriaService.tsx

```diff
@@ -0,0 +1,41 @@
+import { useState, useEffect } from 'react';
```

Comment:
I don't have a ton of exposure to the PathwayService but is there anyway we could generalize that and share instead of copying everything here and just changing types?

Reply:
I'm thinking about generic typing the PathwayService module as a solution, but am open to other ideas!